lua/nordic/compatibility.lua

```diff
@@ -0,0 +1,20 @@
+local function compatability(options)
+    -- All backwards compatibility
+
+    -- Log level
+    local level = vim.log.levels.WARN
+
+    if options.transparent_bg then
+        vim.notify('Nordic.nvim: transparent_bg is deprecated, use transparent instead', level)
```

Comment:
Maybe we should use "config.transparent_bg" and "config.transparent" to make it a bit more clear?

Reply:
Definitely: [feat(deprecation): better warning message;](https://github.com/AlexvZyl/nordic.nvim/pull/127/commits/79bd1b54e318c1b2f4ef744551025e46d2d7251b)